apps/demos/Demos/Chat/Overview/Angular/app/app.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="demo-container">
+    <dx-chat [user]="currentUser" [items]="messages" width="760" height="810"></dx-chat>
```

Comment:
I suggest change the approach to setting dimensions for the Chat and use CSS styles for it. 
We will avoid applying inline styles to the root element and make markup clearer.

Also, it's unclear why we should set the width of the Chat. In our scenario we should stretch it according to the available space inside the container.

Reply:
cfdd79b99f6dd442b66563f8ba5fb3c17623fed1
0ac0142f6d2f837ee5c3e8078461e3a66e02b470